lib/common/templates/flutter/class.dart.template

```diff
@@ -0,0 +1,41 @@
+<%
+//
+// Copyright 2019 Alaska Air Group or its affiliates. All Rights Reserved.
```

Comment:
You might want to change this 😉You can put Amazon, or whatever organization you are affiliated with, or just remove it.